sdk/ai/azure-ai-projects/samples/agents/sample_agents_image_input_url.py

```diff
@@ -0,0 +1,93 @@
+# pylint: disable=line-too-long,useless-suppression
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+DESCRIPTION:
+    This sample demonstrates how to use basic agent operations using image url input for the
+    the Azure Agents service using a synchronous client.
+
+USAGE:
+    python sample_agents_image_input_url.py
+
+    Before running the sample:
+
+    pip install azure-ai-projects azure-identity
+
+    Set these environment variables with your own values:
+    1) PROJECT_CONNECTION_STRING - The project connection string, as found in the overview page of your
+       Azure AI Foundry project.
+    2) MODEL_DEPLOYMENT_NAME - The deployment name of the AI model, as found under the "Name" column in 
+       the "Models + endpoints" tab in your Azure AI Foundry project.
+"""
+
+import os, time
+from typing import List, Dict, Any, cast
+from azure.ai.projects import AIProjectClient
+from azure.identity import DefaultAzureCredential
+from azure.ai.projects.models import MessageTextContent, MessageContentBlockInput
+
+
+# [START create_project_client]
+project_client = AIProjectClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    conn_str=os.environ["PROJECT_CONNECTION_STRING"],
+)
+# [END create_project_client]
+
+with project_client:
+
+    # [START create_agent]
+    agent = project_client.agents.create_agent(
+        model=os.environ["MODEL_DEPLOYMENT_NAME"],
+        name="my-assistant",
+        instructions="You are helpful assistant",
+    )
+    # [END create_agent]
+    print(f"Created agent, agent ID: {agent.id}")
+
+    # [START create_thread]
+    thread = project_client.agents.create_thread()
+    # [END create_thread]
+    print(f"Created thread, thread ID: {thread.id}")
+
+    # [START create_message]
+    image_url = "https://upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Gfp-wisconsin-madison-the-nature-boardwalk.jpg/2560px-Gfp-wisconsin-madison-the-nature-boardwalk.jpg"
```

Comment:
Why not use the raw URL to the image_file.png you have in this GitHub repo? It's a public link. That's what we do in Inference SDK sample for input upload.